packages/ui-dom-utils/src/findDOMNode.ts

```diff
@@ -30,10 +30,14 @@ import ReactDOM from 'react-dom'
  * React 18 (element.ref) and React 19+ (element.props.ref)
  */
 function getElementRef(elem: { ref?: any; props?: { ref: any } }) {
-  if (elem?.props?.ref !== undefined) {
-    return elem.props.ref
+  // Note that this is added by InstUI components like Position
+  if (elem.ref) {
+    return elem.ref
   }
-  return elem.ref
+  // note that trying to access this will display a warning in React 18.
+  // this is OK because without it React itself would show an error because
+  // its using findDOMNode
+  return elem?.props?.ref
 }
```

Comment:
This is the main change. The issue was that BreadCrumb example was showing "ContextView: `ref` is not a prop" warnings